setup.py

```diff
@@ -64,10 +64,22 @@ def setup_path(path = ""):
         install_requires.append('pywin32')
 
     packages = ["pywinauto", "pywinauto.tests", "pywinauto.controls", "pywinauto.linux"]
+elif sys.platform == 'darwin':
+    install_requires = [
+        'codecov', 

```

Comment:
`codecov` is a requirement for running tests with coverage (for development only). `setup.py` must contain only production needed packages.